src/app/project/board/board.component.html

```diff
@@ -13,54 +13,175 @@
-            </span>
-        </div>
-        <div class="app-toolbar-row">
-            <h2 class="mat-title">
-                {{project.longname}}
-            </h2>
-            <span class="fill-remaining-space"></span>
-            <span class="app-toolbar-action">
-                <button mat-icon-button class="right"
-                    (click)="openDialog('editProject', project.shortname, project.shortcode)"
-                    *ngIf="projectAdmin && project.status">
-                    <mat-icon>edit</mat-icon>
-                </button>
-            </span>
+            </div>
+            <div class="app-toolbar-row">
+                <h2 class="mat-title">
+                    {{ project.longname }}
+                </h2>
+                <span class="fill-remaining-space"></span>
+                <span class="app-toolbar-action">
+                    <button mat-icon-button
+                            *ngIf="projectAdmin && project.status"
+                            (click)="openDialog('editProject', project.shortname, project.shortcode)"
+                            class="right">
+                        <mat-icon>edit</mat-icon>
+                    </button>
+                </span>
+            </div>
         </div>
     </div>
 
-    <!-- description -->
-    <section class="project description" *ngFor="let description of project.description">
-        <div [innerHtml]="description.value"></div>
-    </section>
+    <div class="project-info-container flex-panel">
 
-    <mat-divider *ngIf="project.keywords.length > 0"></mat-divider>
+        <div class="left-info column">
+            <!-- description -->
+            <section class="project description" *ngFor="let description of project.description">
+                <h3>Description</h3>
+                <div [innerHtml]="projectDescription"></div>
+            </section>
 
-    <!-- keywords -->
-    <section class="project keywords">
-        <mat-chip-list>
-            <mat-chip *ngFor="let k of project.keywords">{{k}}</mat-chip>
-        </mat-chip-list>
-    </section>
+            <section>
+                <mat-tab-group animationDuration="0ms" [color]="color" class="tab-bar margin-from-top">
+                    <mat-tab>
+                        <ng-template mat-tab-label>
+                            <mat-icon class="tab-icon">perm_data_setting</mat-icon>
+                            Project
+                        </ng-template>
+                        <app-project-tab-view [metadata]="projectMetadata"></app-project-tab-view>
+                    </mat-tab>
 
+                    <mat-tab>
+                        <ng-template mat-tab-label>
+                            <mat-icon class="tab-icon">collections</mat-icon>
+                            Dataset
+                        </ng-template>
+                        <app-dataset-tab-view [metadata]="datasetMetadata" [noOfDatasets]="datasetOptions.length"></app-dataset-tab-view>
```

Comment:
to check: my code editor indicates that "Identifier 'length' is not defined. 'object' does not contain such a member"

Reply:
Fixed.